### ISSUE TITLE
Transfer Ownership of Projects - ReSourcery & Rios Limpios

### ISSUE THREAD

**Issue by Danibelle** [Giveth/giveth-next#645]
These two projects sprang from the Diamante Bridge Collective as a Trace, and are now their own entity!

Please transfer the ownership of these projects from this address: 0x7554f10Da3Ed7128300577e55abCd8F8835BCee4

To:

Resourcery - 0xA36a5453Be6Ef6f3739E8A9B0C656B328Ae099f3
Rios Limpios - 0x2E6FdaC4f7778e902cC6cf632f58bc20AfA8718F

![image](https://user-images.githubusercontent.com/39466970/149669362-faaad322-aa4a-42f5-8eff-2f3098c308bb.png)

## 🎉 Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍ Reviewer

@

**Comment by MoeNick** [comment]
@Danibelle 
Just to make it clear, you mean **the recipient** of the below project should change like this: 

1- ReSourcery  https://giveth.io/project/RECenter-the-Resource-Education-Center-San-Salvador-0   ---> 0xA36a5453Be6Ef6f3739E8A9B0C656B328Ae099f3

2- Rios Limpios https://giveth.io/project/Rios-Limpios-Neighborhood-Grey-Water-Filtration-0  ---> 0x2E6FdaC4f7778e902cC6cf632f58bc20AfA8718F 

So we're not going to change "Project Managers" of these projects, right?

**Comment by Danibelle** [comment]
No these addresses are already the project recipient - but they were created by the DBC account.
We want to give the project recipient the ability to edit and update their own project.

**Comment by MoeNick** [comment]
So please would you let these 2 addresses create an account (and preferably complete the profile ) so we can be able to move the projects to them. @Danibelle

**Comment by Danibelle** [comment]
Ah ha yes thank you!!

**Comment by Danibelle** [comment]
DBC has a [post on their forum](https://forum.diamantebridge.org/t/resourcery-and-rios-limpios-project-owner-change/48/3) about this decision to track the ownership change.

**Comment by Danibelle** [comment]
@MoeNick the profiles have been created.

**Comment by MoeNick** [comment]
@CarlosQ96 can you jump into this task pls?

**Comment by CarlosQ96** [comment]
Yes, I'll do it. @MoeNick Question tho. Should we add something to the admin bro to allow ownership transfer? So if any other projects needs this, anyone of the verification team can do it.

**Comment by CarlosQ96** [comment]
@MoeNick @Danibelle I transferred the projects to the new users that use the new addresses respectively in our db. I double checked Giveth.io and it seems to be ok.
Please double check and let me know!!!
